src/index.html

```diff
@@ -11,7 +11,127 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <div class="wrapper">
+      <h1>HTML Form</h1>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset>
+          <legend>Personal information</legend>
+  
+          <label class="form-field">
+            Surname:
+            <input type="text" name="surname" 
+            autocomplete="off">
+          </label>
+  
+          <label class="form-field">
+            Name:
+            <input type="text" name="name"
+            autocomplete="off" required>
+          </label>
+  
+          <label class="form-field">
+            How old are you?
+            <input type="number" name="age" 
+            min="1" max="100" 
+            value="12">
+          </label>
+  
+          <label class="form-field">
+            Full date of birth:
+            <input type="date" name="birth">            
```

Comment:
remove unnecessary white spaces after elements

```suggestion
            <input type="date" name="birth">
```